lib/node_modules/@stdlib/stats/base/dists/logistic/ctor/benchmark/benchmark.js

```diff
@@ -491,18 +579,23 @@ bench( pkg+':pdf', function benchmark( bm ) {
 	var dist;
 	var mu;
 	var s;
+	var len;
 	var x;
 	var y;
 	var i;
 
 	mu = 2.0;
 	s = 3.0;
 	dist = new Logistic( mu, s );
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( 0.0, 1.0 );
```

Comment:
```suggestion
		x[ i ] = uniform( -3.0, 3.0 );
```
Applies throughout the PR where necessary